.github/workflows/push-code.yml

```diff
@@ -40,6 +40,10 @@ on:
         description: 'Additional options to pass to composer when removing dev dependencies'
         type: string
         default: '-o --apcu-autoloader'
+      dry_run:
```

Comment:
This is never used, what purposes does it play? Wouldn't it be better to remove from the call?

I expect your calling code has it as a carry over from the blt build, where the step also did the build itself?

Reply:
On line 77 of this file: 

```
      DRY_RUN: ${{ inputs.dry_run == 'true' && '--dry-run' || '' }}
```

This was causing errors - it wouldn't even start the build, because there was no `inputs.dry_run` variable set.

I'm quite happy to remove it if you'd prefer - but if it's a leftover from BLT then there may be knock-on effects on the sites that still use that workflow

Reply:
As discussed in Slack - removed the input & environment variables.